File: Lesson_1_Engurazov/app/src/main/java/com/cloniamix/lesson_1_engurazov/screens/MainActivity.java
package com.cloniamix.lesson_1_engurazov.screens;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.cloniamix.lesson_1_engurazov.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void startNewActivity(View view){
        switch (view.getId()){
            case R.id.btn_screen_1:
                startActivity(new Intent(this, Activity1.class));
                break;

            case R.id.btn_screen_2:
                startActivity(new Intent(this, Activity2.class));
                break;
        }
    }
}

File: Lesson_11_Engurazov_Kotlin/app/src/main/java/com/cloniamix/lesson11engurazovkotlin/MainActivity.kt
package com.cloniamix.lesson11engurazovkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        statisticsView.setDayStatistics(getStatistics())

        statisticsView.setOnClickListener { statisticsView.startMyAnimate() }
    }

    override fun onResume() {
        super.onResume()
        statisticsView.startMyAnimate()
    }

    private fun getStatistics(): ArrayList<StatisticsView.DayStatistic> {
        val statistics: ArrayList<StatisticsView.DayStatistic> = ArrayList()

        statistics.add(StatisticsView.DayStatistic("05.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("16.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("18.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("20.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("21.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("22.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("23.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("25.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("26.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("29.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("30.05", (Math.random() * 100).toInt()))
        statistics.add(StatisticsView.DayStatistic("31.05", (Math.random() * 100).toInt()))
        return statistics
    }
}
